samples/texture_3d.html

```diff
@@ -152,16 +186,18 @@
         gl.texImage3D(
             gl.TEXTURE_3D,  // target
             0,              // level
-            gl.R32F,        // internalformat
+            gl.RGB8,        // internalformat
```

Comment:
Also, you could add a comment indicating that the sample was changed from R32F to R8 for best portability, because not all devices can render to floating-point textures (and, further, this functionality is in a WebGL extension: EXT_color_buffer_float), and renderability is a requirement for generating mipmaps.